Jenkinsfile

```diff
@@ -717,12 +717,14 @@ pipeline {
                     }
                     steps {
                         functionalTest inst_repos: daosRepos(),
-                                       inst_rpms: functionalPackages(1, next_version),
+                                       inst_rpms: functionalPackages(1, next_version) + " daos-debuginfo",
```

Comment:
Is this a package that `functionalPackages()` should be adding for any/every stage that uses valgrind?

Reply:
No, I believe this is because the particular stage tests rpms where unit tests/nlt work from the source build.

Reply:
> No, I believe this is because the particular stage tests rpms where unit tests/nlt work from the source build.

Right.  But `functionalPackages()` is only used for test stages that install RPMs.  So the question becomes would any stage that installs RPMs and use valgrind need daos-debuginfo?

Reply:
Yes.  When we add more stages that use valgrind we can resolve this then.

Reply:
> Yes. When we add more stages that use valgrind we can resolve this then.

But in the meanwhile this continues to unnecessarily grow the size of our Jenkinsfile, which we know is limited, until it becomes a fire for the CI team to try to refactor down again.  We simply do not have the resources to kick this kind of can down the road for the resource-constrained CI/release-engineering team to have to put aside release-critical tasks to pick up and resolve.